docs/_includes/documentation-menu.html

```diff
@@ -21,6 +21,7 @@
 				<ul>
 					<li><a href="{{docu}}/features/bindings/astro/readme.html">Astro</a></li>
 					<li><a href="{{docu}}/features/bindings/digitalstrom/readme.html">digitalSTROM</a></li>
+				    <li><a href="{{docu}}/features/bindings/dmx/readme.html">DMX</a></li>
```

Comment:
Could you please replace the spaces by tabs, so that it is aligned with the other rows here?